tcs-client/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.transformuk.hee.tis</groupId>
   <artifactId>tcs-client</artifactId>
-  <version>6.7.1</version>
+  <version>6.7.2</version>
```

Comment:
Risk here: other services using tcs-client needs to be updated. i.e. generic upload, ESR?